### ISSUE TITLE
Loss could not converge when training from scratch

### ISSUE THREAD

**Issue by anti-machinee** [deepinsight/insightface#2017]
My experiment is lauched with information
- GPU: 2
- Model: IResNet50
- Marginal softmax: ArcFace (s=30, m=0.5), do not use partial_fc
- Batchsize : 64*2=128
- Warm up: 0
- Init learning rate: 1e-1
- Loss function: cross entropy
- Dataset: MS1Mv2
But I am facing to some weird case
- Model could not converge and loss function is about 16 in many epoch (It does not like log of Insightface repo, loss seems converge so fast)
Thank you for your attention

**Comment by anxiangsir** [comment]
learning rate should be set lower than 1e-3 when using AdamW

**Comment by Akshaysharma29** [comment]
Hi @anti-machinee I am facing a similar issue my loss is also around 16.
Have you solved this issue?

**Comment by anti-machinee** [comment]
> Hi @anti-machinee I am facing a similar issue my loss is also around 16.
> Have you solved this issue?
> 

You should try to decrease lr to 1e-3 or even 1e-4

**Comment by Akshaysharma29** [comment]
I have tried(1e-4, 1e-5, 1e-6) with adam but my loss didn't decrease.
